### DISCUSSION TITLE
shelly H&T nel congelatore, problema batteria

### DISCUSSION THREAD

**Discussion by issuluca** [indomus/forum#187] [HUB: Home Assistant]
Ho seguito l'articolo con il progetto per monitorare la temperatura del frigo, io lo uso nel congelatore, ho usatto questo dispositivo perché in teoria è uno dei pochi ad essere idoneo per temperature inferiori ai -20°.
Purtroppo ogni 10/15 giorni perde la connessione, immagino sia un problema di batteria.
Sostituisco la batteria e tutto funziona nuovamente.
Qualcuno ha avuto il mio stesso problema?
Come avete risolto?

**Comment by KC-inDomus** [comment]
Io lo uso da tempo senza problemi. Scusa ma hai integrato il sensore di batteria, per capire se effettivamente si scarica?

**Reply by issuluca** [reply]
Si, ma la batteria non si scarica.
Penso che la temperatura crei problemi alla batteria, non tanto che al dispositivo.
Tu che batteria stai usando?

**Reply by KC-inDomus** [reply]
mah, una batteria generica presa su amazon, niente di che
